### ISSUE TITLE
Question on the definition of beam_ids in VisibilitySimulator()

### ISSUE THREAD

**Issue by JianrongTan** [HERA-Team/hera_sim#178]
In the description of `hera_sim.visibilities.simulator.VisibilitySimulator()`, the definition of the parameter `beam_ids` is
```
beam_ids : array_like of int, optional
        List of integers specifying which beam model each antenna
        uses (i.e. the index of `beams` which it should refer to).
        Initialized from `obsparams`, if included. By default, all
        antennas use the same beam (beam 0).
        Shape=(N_ANTS,).
```
It seems `beam_ids` will be `[0,...,0]` if there is only one beam model (beam 0) across all antennas. And I find this definition is consistent with `pyuvsim.simsetup.parse_telescope_params()`.

While in VisCPU(VisibilitySimulator).validate(), we have 

```
if not self.use_pixel_beams:
            for ant in self.ant_list:
                assert len(np.where(self.beam_ids == ant)[0]), (
                    "No beam found for antenna %d" % ant
                )
```
If I understand it correctly, it seems that beam_ids here should contain ant numbers but not beam numbers. Can anyone help to clarify it?

**Comment by steven-murray** [comment]
Hi @JianrongTan, yes I think this is wrong. This is being fixed in #174, which should be merged "soon".

**Comment by JianrongTan** [comment]
Thanks a lot @steven-murray! 
Fixed in #174
